resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -14,48 +14,46 @@
 <div class="container">
     <div class="row">
         <div class="col-sm-6">
-            <center><h4>My Feedback:</h4></center>
-            <div class="table">
-                <table class="table table-bordered">
-                    <thead>
-                        <tr>
-                            <th scope="col"><center>Position</center></th>
-                            <th scope="col"><center>Result</center></th>
-                            <th scope="col"><center>Comments</center></th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        @if($feedback->count() >=1 )
+            @if($feedback->count() > 0)
+                <center><h4>My Feedback:</h4></center>
+                <div class="table">
+                    <table class="table table-bordered">
+                        <thead>
+                            <tr>
+                                <th scope="col"><center>Position</center></th>
+                                <th scope="col"><center>Result</center></th>
+                                <th scope="col"><center>Comments</center></th>
+                            </tr>
+                        </thead>
+                        <tbody>
                             @foreach($feedback as $f)
                                 <tr>
                                     <td><center><a data-toggle="tooltip" title="View Details" href="/dashboard/controllers/profile/feedback-details/{{ $f->id }}">{{ $f->position }}</a></center></td>
                                     <td><center>{{ $f->service_level_text }}</center></td>
                                     <td><center>{{ str_limit($f->comments, 25, '...') }}</center></td>
                                 </tr>
                             @endforeach
-                    </tbody>
-                </table>
-                        @else
-                    </tbody>
-                </table>
-                            <p>No feedback found.</p>
-                        @endif
-               {!! $feedback->links() !!}         
-            </div>
+                        </tbody>
+                    </table>
+                   {!! $feedback->links() !!}         
+                </div>
+            @else 
+                @include('inc.empty_state', ['header' => 'No Feedback', 'body' => 'No feedback found.', 'icon' => 'fa-solid fa-comment'])
+            @endif
```

Comment:
This looks a bit odd when one table is empty and the other has content. Is there any way to align these a bit better since they are side-by-side?
![image](https://github.com/user-attachments/assets/65d96653-e0aa-4788-979a-9f9ef2e5c2bc)

Reply:
Yeah that shouldn't be a problem. I'll take a look

Reply:
Resolved in 20f25cf